python/mpi_cmake_modules/clang_format.py

```diff
@@ -1,16 +1,3 @@
-#!/usr/bin/env python
-""" @namespace mpi_cmake_modules.clang_format
```

Comment:
Why removing the docstring altogether? You could adapt it to say what this file is about...
And if you remove it the documentation will not even display it.

Reply:
this docstring is now part of the executable

Reply:
Yes but then this file do not have any docstring at all. It should have one.

Reply:
+1, will do